build.sbt

```diff
@@ -6,11 +6,11 @@ organization := "com.databricks"
 
 scalaVersion := "2.10.4"
 
-libraryDependencies += "org.apache.spark" %% "spark-sql" % "1.3.0" % "provided"
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "1.4.0"
```

Comment:
Since we now require Spark 1.4+ I wonder if we shouldn't check the `SPARK_VERSION` and throw an error for older versions?  Users on the mailing list are always confused by ClassNotFound or AbstractMethodErrors caused by version mismatches.